source/javascripts/pages/PipelinesPage/components/PipelineCanvas/GraphPipelineCanvas/components/WorkflowNode/Handles.tsx

```diff
@@ -26,6 +39,20 @@ const defaultHandleButtonStyle: CSSProperties = {
   backgroundColor: 'transparent',
 };
 
+const placeholderNode = (dependsOn?: string | null): Node<PipelineWorkflow> => ({
```

Comment:
Maybe `createPlaceholderNode` would be a better name? Same below `createPlaceholderEdge`?